app/src/main/java/com/duckduckgo/app/browser/BrowserPopupMenu.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2018 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.browser
+
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Build.VERSION.SDK_INT
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup.LayoutParams.WRAP_CONTENT
+import android.widget.PopupWindow
+
+class BrowserPopupMenu : PopupWindow {
+
+    constructor(layoutInflater: LayoutInflater, view: View = BrowserPopupMenu.inflate(layoutInflater))
+            : super(view, WRAP_CONTENT, WRAP_CONTENT, true) {
+
+        if (SDK_INT > 21) {
```

Comment:
Is this deliberately doing something different for API 21? I would have thought 21 and up could have elevation supplied (ie, `>= 21`).

If it is deliberately doing something different for Lollipop, then cool. But if it was supposed to be `>= 21` then we can ditch the whole if, given we're `minSDK=21`.